### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by PhucNghia**

### CODE REVIEW

**Comment by PhucNghia** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+/*header*/
+.nav a{
+  color: white !important;
+}
+.navbar{
+  background: #36648B !important;
```
Comment (huongnguyenmta) [inline app/assets/stylesheets/custom.scss]: dùng `variable in Sass`

**Comment by huongnguyenmta** [comment]
commented

**Comment by PhucNghia** [comment]
ready

**Pull request merged**